docs/index.md

```diff
@@ -96,6 +96,17 @@ git checkout v7.0.13
 scramv1 b clean; scramv1 b # always make a clean build
 ```
 
+## Oustide of CMSSW (recommended for non-CMS users)
```

Comment:
Perhaps we should update the name to something like "Precompiled versions with Docker", and perhaps put it under the standalone compilation heading? Otherwise users might get confused and think that other standalone options all require CMSSW.